lib/build.js

```diff
@@ -16,7 +17,13 @@ function do_build(gyp,argv,callback) {
                           concat(['--']).
                           concat(result.unparsed);
         }
+		if (!err && result.opts.napi_build_version) {
+			napi.swap_build_dir_in(result.opts.napi_build_version);
+		}
```

Comment:
You're using tabs but, looking at the surrounding code, it should be spaces.  Happen in a few other places too.